lib/features.js

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+module.exports = config => {
+  const lastAccessTimeUpdates = config.lastAccessTimeUpdates
+  const signinConfirmation = config.signinConfirmation
+
+  return {
+    isSampledUser: isSampledUser,
+
+    isLastAccessTimeEnabled (uid, email) {
+      return lastAccessTimeUpdates.enabled && (
+        isSampledUser(uid, lastAccessTimeUpdates.sampleRate) ||
+        new RegExp(lastAccessTimeUpdates.enabledEmailAddresses).test(email)
+      )
+    },
+
+    isSigninConfirmationEnabled (uid, email, request) {
```

Comment:
> There were test coverage for it, https://github.com/mozilla/fxa-auth-server/blob/master/test/local/account_routes.js#L916

Oh, cool. Fwiw, one of the reasons I like tp have fully mocked-out unit tests for stuff at every layer, as opposed to relying on the route tests to cover everything, is that it can alleviate some of the symptoms discussed in #1448.

Specifically, test failures should become more localised to the specific module that is at fault. I think one of the problems with wading through the error output from `test/local/account_routes.js` is that the watershed for failures is so broad, it's difficult to know where to begin debugging.

So it's a personal mission of mine to gradually mock out more stuff as I go, and try to improve our test isolation in doing so.